app/javascript/mastodon/components/status_content.js

```diff
@@ -162,6 +162,8 @@ export default class StatusContent extends React.PureComponent {
       return (
         <div
           ref={this.setRef}
+          tabIndex='0'
+          aria-label={status.get('search_index')}
```

Comment:
I know this shouldn't be used, but it already contains the perfect information (CW + content), and I would rather avoid additional encodings/decodings.